tests/pytest/test.py

```diff
@@ -1280,6 +1280,87 @@ def testFilterPrecedence(env):
     r.expect('JSON.GET', 'doc', '$[?(@.f==true || @.one==1 && @.t==false)]').equal('[]')
 
 
+def testOutOfRangeValues(env):
+    # Test numeric min/max values
+    r = env
+
+    doc_int_ok = {
+        # i64
+        "max_i64": 9223372036854775807,
+        "beyond_max_i64": 9223372036854775808, # as u64
+        "min_i64": -9223372036854775808,
+        "below_min_i64": -9223372036854775809, # as f64
+        # u64
+        "max_u64": 18446744073709551615,
+        "beyond_max_u64": 18446744073709551616, # as f64
+    }
+
+    doc_float_ok = {
+        "max_f64" : 1.7976931348623157e+308,
+        "max2_f64": 1.7976931348623158e+308,
+        "min_f64" : -1.7976931348623157e+308,
+        "min2_f64": -1.7976931348623158e+308,
+        #"max_u64": 18446744073709551615.0,
+    }
+
+    doc_float_ok_2 = {
+        # i64
+        "max_i64": 9223372036854775807.0,
+        "beyond_max_i64": 9223372036854775808.0, # as u64
+        "min_i64": -9223372036854775808.0,
+        "below_min_i64": -9223372036854775809.0, # as f64
+        # u64
+        "max_u64": 18446744073709551615.0,
+        "beyond_max_u64": 18446744073709551616.0, # as f64
+    }
+
+    doc_bad_values = {
+        "beyond_max_f64": 1.7976931348623159e+308,
+        "below_min_f64": -1.7976931348623159e+308
+    }
+    
+    r.expect('JSON.SET', 'doc_int_ok', '$', json.dumps(doc_int_ok)).ok()
+    r.expect('JSON.SET', 'doc_float_ok', '$', json.dumps(doc_float_ok)).ok()
+    r.expect('JSON.SET', 'doc_float_ok_2', '$', json.dumps(doc_float_ok_2)).ok()
+
+    def check_object_values(obj, name, epsilon):
+        # Test values from JSON.GET are equal to JSON.SET
+        # Check no crash (using get_long internally)
+        arr = []
+        for k, v in iter(obj.items()):
+            r.assertTrue(True, message='GET {}={}'.format(k, v))
```

Comment:
?

Reply:
This is just printing in order to see the current value in the loop

Reply:
We have debugPrint for that
https://github.com/RedisLabsModules/RLTest/blob/09ced571f7d706f39c6a505d84ea2ebb05653b9a/RLTest/env.py#L58

Reply:
It is not enabled by default
`--debug-print         print debug messages (default: False)`